server/src/instant/db/cel.clj

```diff
@@ -170,7 +170,9 @@
   java.util.Map
   (get [_ k]
     (stringify
-     (or (get m k) (get m (keyword k)))))
+     (if (contains? m k)
+       (get m k)
+       (get m (keyword k)))))
```

Comment:
You know for `entrySet` below, we may want to do something like: 

Maybe we should have a function like `(get-cel-value m k)`

Then, for entrySet we can use it too: 

```clojure
(->> (keys (or m {})) 
         (map (fn [k] (get-cel-value m k))
         set) 
```

Reasoning: 

This way when you print the map in your repl, you see the same values you would have seen if you used `get`